runtime/common/src/main/java/edu/snu/onyx/runtime/common/plan/physical/TaskGroup.java

```diff
@@ -27,53 +27,31 @@
  * A stage contains a list of TaskGroups whose length corresponds to stage/operator parallelism.
  */
 public final class TaskGroup implements Serializable {
```

Comment:
Probably unrelated to this PR, but it'd be nice if this class can be removed.

Its variables can be removed/moved like the following
* stageId -> remove, and instead provide PhysicalStage#getId
* taskDAG -> move into PhysicalStage
* containerType -> probably better to be tracked separately in a scheduler component

Can you file an issue, if this makes sense?

Reply:
I like it. I'll file the issue.